jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererClientAdaptorTest.kt

```diff
@@ -110,6 +118,13 @@ class CodeWhispererClientAdaptorTest {
             } doReturn authManagerRule.createConnection(ManagedSsoProfile("us-east-1", aString(), emptyList())) as AwsBearerTokenConnection
         }
         projectRule.project.replaceService(ToolkitConnectionManager::class.java, connectionManager, disposableRule.disposable)
+
+        isTelemetryEnabledDefault = AwsSettings.getInstance().isTelemetryEnabled
+    }
+
+    @After
+    fun tearDown() {
+        AwsSettings.getInstance().isTelemetryEnabled = isTelemetryEnabledDefault
```

Comment:
Is this required? Are we changing the original value anywhere in the tests?

Reply:
yes here: 
```
AwsSettings.getInstance().isTelemetryEnabled = true
        mockApiCall()
        verify(bearerClient).sendTelemetryEvent(
            argThat<SendTelemetryEventRequest> { optOutPreference() == OptOutPreference.OPTIN }
        )

        AwsSettings.getInstance().isTelemetryEnabled = false
```